src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1409,6 +1409,16 @@ void NetworkOPsImp::switchLastClosedLedger (
     clearNeedNetworkLedger ();
     newLedger->setClosed ();
     auto openLedger = std::make_shared<Ledger> (false, std::ref (*newLedger));
+    // Apply transactions from the current open ledger to
+    // this new one, including locally held transactions.
+    {
+        // VFALCO Which hash do we use?
```

Comment:
I'm not saying that either. I'd like a *deep* review...can you please investigate the usage of the hash in `CanonicalTXSet` constructor, compare against existing uses of `CanonicalTXSet`, and render your opinion on this change?